sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/SqlAnalyzer.java

```diff
@@ -115,6 +116,22 @@ static boolean isEndOfInput(ParseResumeLocation parseResumeLocation) {
         >= parseResumeLocation.getInput().getBytes(UTF_8).length;
   }
 
+  static String getOptionStringValue(
+      ResolvedCreateFunctionStmt createFunctionStmt, String optionName) {
+    for (ResolvedNodes.ResolvedOption option : createFunctionStmt.getOptionList()) {
+      if (option.getName().equals(optionName)) {
```

Comment:
How about `optionName.equals(option.getName())`. That will avoid potential crashes if `getName` returns null.